src/components/Canvas/index.ts

```diff
@@ -1404,6 +1404,11 @@ export default class Canvas extends EventDispatcher {
     const mouseCartCoord = this.getMouseCartCoord(evt);
     const pixelIndex = this.getPixelIndexFromMouseCartCoord(mouseCartCoord);
     if (pixelIndex) {
+      const cursorStyle = `url("/cursor/${this.brushMode}.cur"), auto`;
+      if (cursorStyle !== this.element.style.cursor) {
+        this.element.style.cursor = cursorStyle;
+      }
+
```

Comment:
Instead of creating an if else statement inside onMouseMove and render the cursor, why don't we create a `cursorStyle` member variable and a `setCursorStyle` method (or simply create `setCursorSvg` method, and create a cursor style internally in the function. We could also allow users to input their own svg icons to the dotting prop, allowing users to style their own cursors also.

An example that uses svg element for styling cursor is shown in [react-artboard](https://github.com/ascorbic/react-artboard/blob/894a406553d8f85438f60070aa25dbe36df27dbd/src/utils/cursors.ts) Similar to the implementation shown in react-artboard, we could allow users to input svg icons to dotting props.